threshold: 0.5 
classes_by_category:
    people: [person]
classes:
    - person
model_file: yolov5s-pose.pt

subscribers:
  
  queue_size: 1
  exact_time: false
  slop: 0.2

  image_rgb: /butia_vision/bvb/image_rgb

  points: /butia_vision/bvb/points

publishers:

  pose_recognition:
    topic: /butia_vision/br/pose_recognition
    queue_size: 1

servers:

  list_classes:
    service: /butia_vision/br/pose_recognition/list_classes

  start:
    service: /butia_vision/br/pose_recognition/start

  stop:
    service: /butia_vision/br/pose_recognition/stop
